ebr_trackerbot/storage/odbc.py

```diff
@@ -0,0 +1,44 @@
+"""
+Slack Bot ODBC Storage
+"""
+
+import pyodbc
+import logging
+from bot import register_storage, config
+import db
+from functools import partial
+
+
+def create_connection(config):
+    if config.get("odbc_connection_string") is None:
+        raise RuntimeError("Missing odbc_connection_string in configuration")
+    link = pyodbc.connect(config.get("odbc_connection_string"))
+    db.create_table(lambda: link)
+    return link
+
+
+def get_connection(config):
+    """
+    Retrieve odbc connection
+    """
+    if get_connection.link is None:
+        get_connection.link = create_connection(config)
+    try:
+        cursor = get_connection.link.cursor()
+        cursor.close()
+    except:
+        get_connection.link = create_connection(config)
+    return get_connection.link
+
+
+get_connection.link = None
```

Comment:
Where is this object coming from?

Reply:
Which object?

Reply:
get_connection

Reply:
it's a static variable for get_connection function.